byzcoin/transaction.go

```diff
@@ -345,11 +345,12 @@ func (instrs Instructions) Hash() []byte {
 type TxResults []TxResult
 
 // NewTxResults takes a list of client transactions and wraps them up
-// in a TxResults with Accepted set to false for each.
+// in a TxResults with Accepted set to true for each.
 func NewTxResults(ct ...ClientTransaction) TxResults {
 	out := make([]TxResult, len(ct))
 	for i := range ct {
 		out[i].ClientTransaction = ct[i]
+		out[i].Accepted = true
```

Comment:
Why? The default of false was more conservative.

Reply:
I'm not sure anymore why I put this to `true` :( I looked at all occurrences of `NewTxResults`, and it's always when calling `createStateChanges`, which sets the `Accepted` field anyway, so it doesn't change whether it's `true` or `false` here...